jbpm-workitems/jbpm-workitems-bpmn2/pom.xml

```diff
@@ -78,6 +78,12 @@
       <groupId>org.apache.ws.xmlschema</groupId>
       <artifactId>xmlschema-core</artifactId>
       <scope>runtime</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.bcel</groupId>
+          <artifactId>bcel</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
I tried out to run a full build, including tests on jdk11 without this exclusion and works fine. So probably this is not needed? wdyt?

Reply:
OK, removed, thanks for your review @afalhambra !